front/src/pages/Plans.jsx

```diff
@@ -1,13 +1,35 @@
-import React, { useState, useEffect } from 'react';
-
+import React, {useState, useEffect} from 'react';
 
 export const Plans = () => {
 
+    function sayHello() {
+        console.log("Hello");
+    }
+
     return (
         <div className="container">
-            <div className="container__content">
-                <h1>Plans</h1>
-            </div>
+            <h1>Plans</h1>
+
+            <br/><br/>
+
+            <button className="btn btn--primary btn--sm" onClick={sayHello}>Default</button>
```

Comment:
There should be one <Button/> component - which can get property like:
-disabled
-className
-onClick